### DISCUSSION TITLE
Commercial Projects on personal/free plan?

### DISCUSSION THREAD

**Discussion by avdain** [supabase/supabase#21044] [Questions] [answered]
Hi, I am writing to inquire about the use of **commercial projects** on the personal/free plan.

Could someone please clarify if this is allowed under the **terms of use**? If not, what are the specific limitations regarding commercial use?

Thanks a lot for your time and help.

**Comment by mwoss** [comment, marked as answer]
Yes, as far as I know, it's allowed to run commercial projects on the Supabase free tier. You have to keep in mind all the limitations of the free tier like no backups, automatic project pausing if not used, lack of custom domains, only 1-day log retention, etc.
